lib/_007/Parser/Syntax.pm

```diff
@@ -182,6 +182,13 @@ grammar _007::Parser::Syntax {
     token term:array { '[' ~ ']' [[<.ws> <EXPR>]* %% [\h* ','] <.ws>] }
     token term:str { <str> }
     token term:parens { '(' ~ ')' <EXPR> }
+    token term:regex {
+        '/' ~ '/'
+        [
+            { check-feature-flag("regex syntax", "REGEX"); }
```

Comment:
Should have a capital `R` in `Regex syntax`, I believe. The `$feature` occurs first in the error message that you get when you don't have the flag turned on:

    $ bin/007 -e='say(/foo/)'
    regex syntax is experimental and requires %*ENV<FLAG_007_REGEX> to be set
    [...]